src/lib/Libifl/Makefile.am

```diff
@@ -0,0 +1,46 @@
+#
+# Copyright (C) 1994-2020 Altair Engineering, Inc.
+# For more information, contact Altair at www.altair.com.
+#
+# This file is part of the PBS Professional ("PBS Pro") software.
+#
+# Open Source License Information:
+#
+# PBS Pro is free software. You can redistribute it and/or modify it under the
+# terms of the GNU Affero General Public License as published by the Free
+# Software Foundation, either version 3 of the License, or (at your option) any
+# later version.
+#
+# PBS Pro is distributed in the hope that it will be useful, but WITHOUT ANY
+# WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+# FOR A PARTICULAR PURPOSE.
+# See the GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+# Commercial License Information:
+#
+# For a copy of the commercial license terms and conditions,
+# go to: (http://www.pbspro.com/UserArea/agreement.html)
+# or contact the Altair Legal Department.
+#
+# Altair’s dual-license business model allows companies, individuals, and
+# organizations to create proprietary derivative works of PBS Pro and
+# distribute them - whether embedded or bundled with other software -
+# under a commercial license agreement.
+#
+# Use of Altair’s trademarks, including but not limited to "PBS™",
+# "PBS Professional®", and "PBS Pro™" and Altair’s logos is subject to Altair's
+# trademark licensing policies.
+#
+
+BUILT_SOURCES = pbs_ifl_wrap.c
+EXTRA_DIST = pbs_ifl.i
+
+pbs_ifl_wrap.c: pbs_ifl.i
+	@swig_dir@/bin/swig -python -outcurrentdir -I$(top_srcdir)/src/include $^
```

Comment:
I think we need to include `@swig_py_inc@` in this command

Reply:
@toonen : I agree with Hiren here too. WIthout the @swig_py_inc, pbs_ifl_wrap.c is not getting created. It errors out with "Error: Unable to find 'python.swg". I actually had to add it in order to make it work.